index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700&family=Roboto+Serif:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header class="head">
+      <h3><a href="index.html">My Logo</a></h3>
+      <nav>
+        <img id="union" src="images/Union.png" />
+      </nav>
+    </header>
+    <main>
+      <section class="home">
+        <div class="headline">
+          <h1>Hey There. I'm Cosmas</h1>
+          <h2>I'm a software developer</h2>
+          <p>
+            I can help you build a proffessional website, feature. Look through
+            some of my work and experience and if you are pleased with my work,
+            contact me.
+          </p>
+        </div>
+        <div class="social-icons">
+          <ul>
+            <li><img src="images/Vector sv1.svg" /></li>
+            <li><img id="vec" src="images/Group icon.svg" /></li>
+            <li><img src="images/Vector sv4.svg" /></li>
+            <li><img src="images/Vector sv5.svg" /></li>
+            <li><img src="images/Vector sv6.svg" /></li>
+          </ul>
```

Comment:
- [Optional] Great job for using `ul` and `li` for social icons 👍 I think it would be better if you wrap each social icon with an `a` tag because Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.👌